dm/syncer/dml.go

```diff
@@ -479,6 +492,7 @@ func newDML(op opType, safeMode bool, targetTableID string, sourceTable *filter.
 		originOldValues:     originOldValues,
 		originValues:        originValues,
 		downstreamIndexInfo: downstreamIndexInfo,
+		downstreamTableInfo: downstreamTableInfo,
```

Comment:
How about rename downstreamIndexInfo to distinguish downstreamIndexInfo and downstreamTableInfo.indexCache?

Reply:
How about `pickedDownstreamIndexInfo`..